packages/protocol/contracts/psp/AirnodePsp.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+contract AirnodePsp {
+    mapping(address => address) public airnodeToAuthorizer;
+    mapping(address => mapping(address => bool))
+        public sponsorToSubscriberToSponsorshipStatus;
+    mapping(address => uint256) public subscriberToSubscriptionCountPlusOne;
+
+    function setAuthorizer(address authorizer) external {
+        airnodeToAuthorizer[msg.sender] = authorizer;
```

Comment:
Is msg.sender the airnode operator? does this mean that an airnode has authorizers by operator instead of global authorizers like in RRP?

Reply:
Here it's the `airnodeAddress` calling this function for simplicity. We can also implement some adminship functionality where `airnodeAddress` can authorize an admin address to call this function, but that's not needed yet.